Bottom: fbda33be55a92f6aa7ed6296172ae7b366741ce9
Top:    8697b6c85cf5ce89fb64e98f454c119ffa063d51
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-04 22:58:19 +0000

Refresh of add-cache-for-board-outline

---

diff --git a/src/polygon.c b/src/polygon.c
index bb33330..39b2253 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -1653,7 +1653,7 @@ ClearFromPolygon (DataType * Data, int type, void *ptr1, void *ptr2)
     }
 
   if (type == POLYGON_TYPE)
-    InitClip (PCB->Data, (LayerType *)ptr1, (PolygonType *) ptr2);
+    InitClip (PCB->Data, (LayerType *) ptr1, (PolygonType *) ptr2);
   else
     PlowsPolygon (Data, type, ptr1, ptr2, subtract_plow);
 }
